lib/turn.rb

```diff
@@ -0,0 +1,29 @@
+
+class Turn
+  attr_reader   :guess
+                :card
+
+  def initialize(guess, card)
+    @guess      = guess
+    @Card       = card
+  end
+
+  def guess
+    @guess
+  end
+
+  def card
+    @card
+  end
+
+  def correct?
+    guess == card.answer
+  end
+
+  def feedback
+    if guess == card.answer
```

Comment:
To DRY up your code, on line 24 I would use the method you created otherwise, why did you create it? 

if correct?
...